packages/sdk-client/src/domain/domain-interface.ts

```diff
@@ -38,11 +38,13 @@ export interface ApplicationCredentials {
 export interface ApiHostname {
   authHostname?: string;
   conversationHostname?: string;
+  templatesHostname?: string;
```

Comment:
`conversationTemplatesHostName` could have more sense :
- help to identify it is related to conversation domain 
- avoid naming clash and confusion in case other kind of templates will have to be managed in the future for a different domain

(same "good" logic as when `voiceApplicationManagementHostname` was named)